shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1312,15 +1313,15 @@ public static WireCommand readFrom(DataInput in, int length) throws IOException
         final String source;
         @ToString.Exclude
         final String delegationToken;
-        final List<ConditionalAttributeUpdate> attributeUpdates;
+        final Optional<BatchInfo> batch;
```

Comment:
`BatchInfo` is a nullable type. Please do not use `Optional` for nullable types ; `null` should signify its inexistence.

Reply:
Ok. fixed.